File: yizhigou-manager-web/src/main/webapp/js/service/brandService.js
app.service("brandService",function($http){
    //查询单条
    this.findOne=function(id){
        return  $http.get("../brand/findOne.do?id="+id);
    }
    //删除
    this.dele=function(ids){
        return $http.get('../brand/delete.do?ids='+ids);
    }
    //添加
    this.add=function(entity){
        return  $http.post('../brand/add.do',entity);
    }
    //修改
    this.update=function(entity){
        return  $http.post('../brand/update.do',entity);
    }

    //查询
    this.search=function(entity,page,rows){
        return $http.post('../brand/search.do?page='+page+"&rows="+rows,entity);
    }

    //查询品牌列表
    this.selectOptionList=function(){
        return $http.get("../brand/selectOptionList.do");
    }

})

File: yizhigou-sellergoods-interface/src/main/java/com/yizhigou/sellergoods/service/BrandService.java
package com.yizhigou.sellergoods.service;

import com.yizhigou.pojo.TbBrand;
import entity.PageResult;

import java.util.List;
import java.util.Map;

public interface BrandService {

    public List<TbBrand>  getBrandList();

    PageResult findPage(TbBrand tbBrand,int pageNum,int pageSize);

    void  save(TbBrand brand);

    TbBrand findOne(long  id);

    void  update(TbBrand tbBrand);

    void delete(Long[] id);

    List<Map>  selectOptionList();

}
